cmd/vic-machine/create/create.go

```diff
@@ -1610,6 +1610,11 @@ func splitVnetParam(p string) (vnet string, value string, err error) {
 		return
 	}
 
+	if strings.Contains(vnet, " ") && (len(mapped) == 1 || (len(mapped) == 2 && len(mapped[1]) == 0)) {
```

Comment:
I think the cause of this is that we can't have a space in the guestinfo. Though the line could use a comment to explain what we're checking for here.